app/views/membership_application_mailer/new_application.html.haml

```diff
@@ -0,0 +1,6 @@
+Hi #{@admin.first_name},
+
+%p A new application has been submitted by #{@membership_application.user_full_name}, with company number: #{@membership_application.company_number} and company id: #{@membership_application.company_id}.
+
+%br Regards,
+%em= ENV['SHF_ADMIN_EMAIL']
```

Comment:
This should probably make use of translations and preferably make use of the ShfMailer templates to encourage a uniform look and feel for all emails.  

The `#{@membership_application.user_full_name}` can be replaced with `#{member_full_name}` after adding the `MembershipApplicationsHelper` to the mailer class (see above)